src/config.js

```diff
@@ -145,11 +145,11 @@ export default class Config {
 
   nonInteractive: boolean;
 
-  workspacesExperimental: boolean;
+  workspacesEnabled: boolean;
 
   //
   cwd: string;
-  worktreeFolder: ?string;
+  workspaceFolder: ?string;
```

Comment:
I'd prefer `workspaceRootFolder` so that the implication that the variable is `null` when not using workspaces is made clear. Why not keep `worktreeFolder` as the designated term for the workspace roots? It clear and concise! 😛